### PR TITLE
chore(Example): Add auth effects marble tests

### PR DESCRIPTION
**Pull request by bcabanes**
Hi!

This PR adds some marble tests for _authEffects_ in the example application.

Feel free to ask any change, available to do so!

### CODE REVIEW

File: example-app/app/auth/effects/auth.effects.spec.ts [hunk 1]
```diff
@@ -0,0 +1,82 @@
+import { TestBed } from '@angular/core/testing';
+import { Actions } from '@ngrx/effects';
+import { RouterTestingModule } from '@angular/router/testing';
+import { cold, hot } from 'jasmine-marbles';
+import { Observable } from 'rxjs/Observable';
+import 'rxjs/add/observable/empty';
```
Comment (brandonroberts) [inline example-app/app/auth/effects/auth.effects.spec.ts]: Use `import { empty } from 'rxjs/observable/empty';` instead of patching the Observable

**Comment by bcabanes** [comment]
@brandonroberts I added the tests and fix the comments!

File: example-app/app/auth/effects/auth.effects.spec.ts [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+
+      actions$.stream = hot('-a---', { a: action });
+      const response = cold('-#|', {}, error);
+      const expected = cold('--b', { b: completion });
+      authService.login = jest.fn(() => response);
+
+      expect(effects.login$).toBeObservable(expected);
+    });
+  });
+
```
Comment (brandonroberts) [inline example-app/app/auth/effects/auth.effects.spec.ts]: The two test below don't seem correct. You're not dispatching any actions to trigger login success and `router.navigate` should be called with the string wrapped in an array (e.g. `['/']`)

Comment (bcabanes) [reply example-app/app/auth/effects/auth.effects.spec.ts]: Thank you for the comment!
I dispatch now the action and have verified that `router.navigate` method was indeed called with the right arguments (using `router.navigate.calls.allArgs()`). So no _false positive_ anymore 🙂

**Comment by brandonroberts** [comment]
Thanks!

**Pull request merged**